### PR TITLE
Remove repos from PaymentsController

### PR DESCRIPTION
**Pull request by leogrim**
@ryanpbrewster @camhashemi

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/PaymentsController.scala [hunk 1]
```diff
@@ -97,9 +95,8 @@ class PaymentsController @Inject() (
         orgCommander.setAccountFeatureSettings(settingsRequest) match {
           case Left(fail) => fail.asErrorResponse
           case Right(response) =>
-            val plan = db.readOnlyMaster { implicit session => paidPlanRepo.get(paidAccountRepo.getByOrgId(request.orgId).planId) }
-            val result = ExternalOrganizationConfiguration(plan.showUpsells, OrganizationSettingsWithEditability(response.config.settings, plan.editableFeatures))
-            Ok(Json.toJson(result))
+            val config = orgCommander.getExternalOrgConfiguration(request.orgId)
+            Ok(Json.toJson(config))
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/PaymentsController.scala:99]: Is this used as intended?

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/PaymentsController.scala:99]: This is going to run into issues with the `readOnlyReplica` lag. Better to convert the `response.config` to an ExternalConfig.

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/PaymentsController.scala:99]: Ah, I see that's what it was doing. Do you have a strong preference for removing repos from controllers?

As a compromise, I might suggest having a method `OrganizationConfiguration => ExternalOrganizationConfiguration` (maybe in the commander? I might slightly prefer isolating this from the rest of the logic in the commander).

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/PaymentsController.scala:99]: Well the code in the controller is duplicating the commander's

**Pull request merged**